core-addon/IonCore.js

```diff
@@ -17,6 +17,6 @@ module.exports = class IonCore {
   }
 
   _openControlPanel() {
-    chrome.runtime.openOptionsPage();
+    browser.runtime.openOptionsPage();
```

Comment:
This returns a promise that might reject, do you want to either catch this (and log it I guess?) or maybe make `initialize` an async function?

Reply:
How about this - could we store a boolean (`false` by default) somewhere (webextension storage probably) which is set to `true` if this page loads successfully? Then on startup of the extension, we can try to display the page again.

I can't think *why* exactly this would fail, but both MDN and the Chrome extension docs mention it: https://developer.chrome.com/extensions/runtime#method-openOptionsPage

Reply:
> How about this - could we store a boolean (`false` by default) somewhere (webextension storage probably) which is set to `true` if this page loads successfully? If it fails, then on startup of the extension, we can try to display the page again.

This seems like a very good idea. Do you mind if I take it to a separate, future PR while only catching it here?

Reply:
wfm!

Reply:
Filed issue #17